release/gcb/cloud_builder.sh

```diff
@@ -75,6 +75,34 @@ add_license_to_tar_images "${REL_DOCKER_HUB}" "${CB_VERSION}" "${OUTPUT_PATH}"
 # log where git thinks the build might be dirty
 git status
 
+#Handle CNI artifacts.
+pushd ../cni
+CNI_OUT=$(make DEBUG=0 where-is-out)
+# CNI version strategy is to have CNI run lock step with Istio i.e. CB_VERSION
+CNI_VERSION=${ISTIO_VERSION}
+MAKE_TARGETS=(build)
+VERBOSE=1 DEBUG=0 ISTIO_DOCKER_HUB="${CB_ISTIOCTL_DOCKER_HUB}" HUB="${CB_ISTIOCTL_DOCKER_HUB}" VERSION="${CNI_VERSION}" TAG="${CNI_VERSION}" make "${MAKE_TARGETS[@]}"
+BUILD_DOCKER_TARGETS=(docker.save)
+
+if [ -d "${CNI_OUT}/docker" ]; then
+    rm -r "${CNI_OUT}/docker"
+fi
+
+VERBOSE=1 DEBUG=0 ISTIO_DOCKER_HUB=${REL_DOCKER_HUB} HUB=${REL_DOCKER_HUB} VERSION="${CNI_VERSION}" TAG="${CNI_VERSION}" make "${BUILD_DOCKER_TARGETS[@]}"
+
+mkdir "/cni_tmp"
+
+cp -r "${CNI_OUT}/docker" "/cni_tmp"
+
+go run ../istio/tools/license/get_dep_licenses.go --branch "master" > LICENSES.txt
```

Comment:
is cni repo going to follow istio's branching strategy? (i.e. a branch for each major release like release-1.0 and release-1.1)

I don't think it is a good idea to hard code master here.

Reply:
Opps that was a left-over I meant to fix after the email discussion.  I will fix it.